Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Applications/Locators/CalculatorWindow.cs

```diff
@@ -1,22 +1,42 @@
-﻿using OpenQA.Selenium;
+﻿using Aquality.WinAppDriver.Elements.Interfaces;
+using Aquality.WinAppDriver.Windows;
+using OpenQA.Selenium;
 using OpenQA.Selenium.Appium;
 
 namespace Aquality.WinAppDriver.Tests.Applications.Locators
 {
-    public static class CalculatorWindow
+    public class CalculatorWindow : Window
     {
         public static By WindowLocator => By.TagName("Window");
 
-        public static By OneButton => By.Name("1");
+        public static By OneButtonLocator => By.Name("1");
```

Comment:
we don't need these locators as separate properties. let's get them from elements:
var calculatorWindow = new CalculatorWindow();
ApplicationManager.Application.Driver.FindElement(calculatorWindow.OneButton.Locator).Click();

Reply:
done

Reply:
done